src/symbolize.h

```diff
@@ -77,6 +77,8 @@
 #  error "symbolize.h" was not included correctly.
 #endif
 
+// INFO(sergiud): Add new features to build system instead of here.
+// See GitHub pull request !1114 for a detail discussion.
```

Comment:
To avoid any confusion we should entirely document the current approach:
```suggestion
// We prefer to let the build system detect the availability of certain features such as symbolization support.
// HAVE_SYMBOLIZE should therefore be defined by the build system in general unless there is a good reason to
// perform the detection using the preprocessor.
```